### ISSUE TITLE
Clean up md files in the lab folders

### ISSUE THREAD

**Issue by florianhartig** [florianhartig/ResearchSkills#29]
- Each link should have a proper descrition, as in \* [Description](URL)
- Add text / explanations where appropriate
- Look generally out for redundancies / inconsistencies

**Comment by Katrin2208** [comment]
Done!

**Comment by florianhartig** [comment]
Hi Katrin,

links sind super aber ich würde gerne noch etwas mehr Ordnung schaffen, deshalb wieder geöffnet. Vielleicht können wir morgen noch mal zusammen draufschauen.

F

**Comment by Katrin2208** [comment]
Klar, gern! Fand es eigentlich schon von vornherein sehr übersichtlich und die Ordner gliedern die Themen auch sinnvoll. Aber können gern noch einmal zusammen drüberschauen
